dashboards: add replica cpu to repl dashboard

kvoli opened the pull request:
In #96127 we added the option to load balance replica CPU instead of QPS across
stores in a cluster. It is desirable to view the signal being controlled for
rebalancing in the replication dashboard, similar to QPS.

This pr adds the `rebalancing.cpunanospersecond` metric to the replication
metrics dashboard.

The avg QPS graph on the replication graph previously described the metric as
"Exponentially weighted average", however this is not true.

This pr updates the description to just be "moving average" which is accurate.
Note that follow the workload does use an exponentially weighted value, however
the metric in the dashboard is not the same.

This pr also updates the graph header to include Replica in the title: "Average
Replica Queries per Node". QPS is specific to replicas. This is already
mentioned in the description.

Resolves: #98109

kvoli committed:
dashboards: add replica cpu to repl dashboard

In #96127 we added the option to load balance replica CPU instead of QPS
across stores in a cluster. It is desirable to view the signal being
controlled for rebalancing in the replication dashboard, similar to QPS.

This commit adds the `rebalancing.cpunanospersecond` metric to the
replication metrics dashboard.

Resolves: #98109

Release note (ui change): `rebalancing.cpunanospersecond` is now
included in the replication metrics dashboard.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/98270)
<!-- Reviewable:end -->

kvoli committed:
dashboards: correct avg qps graph desc

The avg QPS graph on the replication graph previously described the
metric as "Exponentially weighted average", however this is not true.

This commit updates the description to just be "moving average" which is
accurate. Note that follow the workload does use an exponentially
weighted value, however the metric in the dashboard is not the same.

This commit also updates the graph header to include Replica in the
title: "Average Replica Queries per Node". QPS is specific to
replicas. This is already mentioned in the description.

Release note (ui change): The description of the average QPS graph in the
replication metrics dashboard no longer claims the average is
exponentially weighted.

xinhaoz commented on pkg/ui/workspaces/db-console/src/views/cluster/containers/nodeGraphs/dashboards/replication.tsx:
```diff
@@ -102,6 +104,24 @@ export default function (props: GraphDashboardProps) {
+         rebalancing decisions.`}
+    >
+      <Axis units={AxisUnits.Duration} label="CPU time">
+        {_.map(nodeIDs, nid => (
```
nit: prefer native array map over lodash's map

kvoli replied:
Updated to use native map. I also pushed a preceding commit that replaces `_.map` with the native array map for the other graphs in `replication.tsx`.

xinhaoz reviewed (approved):
LGTM with the map swap.

kvoli committed:
dashboards: use native array map instead of lodash

This commit updates the replication dashboard to use the native
javascript array map function, instead of lodash map.

Release note: None

kvoli commented:
TYFTR

bors r=xinhaoz
